activity-handler.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+var os = require("os"),
+    util = require("util");
+
+var async = require("async"),
+    debug = require("debug")("swfr:activity"),
+    humanize = require("humanize-plus");
+
+var activity = require("./activity"),
+    activities = require("./lib/activities");
+
+var activityHandler = function activityHandler(activitiesFolder, id) {
+  id = id | 1;
```

Comment:
Not sure what `console.log("worker #%d:", id, task);` below would do without it...is there a better default? Or should I just check if it's defined for printing out the worker?